submit_and_compare/views.py

```diff
@@ -8,7 +8,10 @@
 from lxml import etree
 from six import StringIO
 from xblock.core import XBlock
-from xblockutils.resources import ResourceLoader
+try:
+    from xblock.utils.resources import ResourceLoader
+except ModuleNotFoundError:  # For backward compatibility with releases older than Quince.
+    from xblockutils.resources import ResourceLoader
```

Comment:
Would this be removed in future?

Reply:
not near, perhaps once we will deprecate all open edX releases before Quince